pyuvsim/simsetup.py

```diff
@@ -1719,6 +1706,12 @@ def initialize_uvdata_from_params(
 
     uv_obj = UVData()
     uv_obj._set_future_array_shapes()
+
+    # Setting the frame and acceptable range for the moon
+    uv_obj._telescope_location.frame = uvparam_dict['telescope_frame']
+    if uv_obj._telescope_location.frame == 'mcmf':
+        uv_obj._telescope_location.acceptable_range = (1.71e6, 1.75e6)
```

Comment:
I don't think these lines are necessary any more, I think this is done by pyuvdata when you set the frame.